CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ list(APPEND CMAKE_MODULE_PATH ${TORCH_XPU_OPS_ROOT}/cmake/Modules)
 include(${TORCH_XPU_OPS_ROOT}/cmake/SYCL.cmake)
 include(${TORCH_XPU_OPS_ROOT}/cmake/BuildFlags.cmake)
 
-option(USE_XCCL "Build with XCCL support" ON)
+option(USE_XCCL "Build with XCCL support" OFF)
```

Comment:
why?

Reply:
Currently, XCCL backend does not get ready in PyTorch release. So we disable it by default (you still can build XCCL with `USE_XCCL`=1) and will enable again in PyTorch 2.8 release by our expectation.

Reply:
That's quite unfortunate since a number of projects awaits distributed support fromXPU. What are outstanding issues preventing to enable XCCL by default?